src/pages/todoSetup/hooks/useTodoSetup/useTodoSetup.js

```diff
@@ -10,29 +10,29 @@ import { createTodoItem } from './useTodoSetup.helpers'
 function useTodoSetup(todoList, updateTodoList, saveTodoList) {
     const [todoText, setTodoText] = useState("");
 
-       const handleTodoTextChange = (e) => {
+       const handleTodoTextChange = useCallback((e) => {
             setTodoText(e.target.value);
-        }
+        },[]);
```

Comment:
[] <> EMPTY_ARRAY
We can provide setTodoText instead of EMPTY_ARRAY as a dependency.